### PR TITLE
(DOCS-4655) Fix up formatting on DD Apps doc

### PR DESCRIPTION
**Pull request by hestonhoffman**
## Motivation

-  Docs bug fix, see [DOCS-4655](https://datadoghq.atlassian.net/browse/DOCS-4655)

## Changes

-   This fixes the formatting on the [Datadog Apps](https://docs.datadoghq.com/developers/datadog_apps/) page and makes some additional style and grammar adjustments. The formatting is making the current page a little hard to follow on the docs site. I don't have a preview of the changes, but I've previewed them locally 👍

Choose one:

-   [x] No release is necessary.
        If you're only updating examples/documentation, this is likely what you want.
-   [ ] All packages that need a release have a changeset.

[DOCS-4655]: https://datadoghq.atlassian.net/browse/DOCS-4655?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/en/getting-started.md [hunk 1]
```diff
@@ -19,97 +19,88 @@ Datadog Apps enable developers to extend the native functionality of Datadog thr
...
-
-
-
-To build your application, run `yarn build` in your terminal. Then move your static generated site to the hosting of your choice and update the URLs in the app settings.
+1. To build your application, run `yarn build` in your terminal. Then move your static generated site to the hosting of your choice and update the URLs in the app settings.
 
 ### OAuth API Access
 
-When this feature is enabled, users need to be authenticated before using the app. This feature allows you to integrate your existing authentication mechanism (for example, cookie-based username/password login) with the Developer Platform.
+When OAuth API Access is enabled, users need to be authenticated before using the app. This feature allows you to integrate your existing authentication mechanism (for example, cookie-based username/password login) with the Developer Platform.
```
Comment (alai97) [inline docs/en/getting-started.md]: Should we add a link to the OAuth API endpoints page: https://docs.datadoghq.com/developers/authorization/oauth2_endpoints/?tab=authorizationendpoints or to this Markdown file (https://github.com/DataDog/apps/blob/master/docs/en/programming-model.md#oauth-api-access)?

```suggestion
When OAuth API Access is enabled, users need to be authenticated before using the app. This feature allows you to integrate your existing authentication mechanism (for example, cookie-based username and password login) with the Developer Platform.
```

File: docs/en/getting-started.md [hunk 2]
```diff
@@ -19,97 +19,88 @@ Datadog Apps enable developers to extend the native functionality of Datadog thr
...
 
 ### Sample Applications
 
-[Starter kit][1]
-
-[Sentiment analysis][2]
+- [Starter kit][1]
+- [Sentiment analysis][2]
```
Comment (alai97) [inline docs/en/getting-started.md:106]:
```suggestion
## Further reading

- [Learn about the Datadog Developer Platform](https://docs.datadoghq.com/developers/authorization/oauth2_in_datadog/)
- [Learn about UI Extensions](https://github.com/DataDog/apps/blob/master/docs/en/programming-model.md#oauth-api-access)
```

File: docs/en/getting-started.md [hunk 3]
```diff
@@ -19,97 +19,88 @@ Datadog Apps enable developers to extend the native functionality of Datadog thr
...
 You may notice an uncaught <strong>HandshakeTimeoutError</strong> in your JavaScript Console when you interact with the local widget in your browser directly. This is expected. The Datadog Apps SDK is <a href="https://github.com/DataDog/apps/blob/master/docs/en/programming-model.md">designed</a> to run in an iframe that connects to the Datadog User Interface and the handshake attempt between the widget and the Datadog UI will timeout when there is no Datadog UI for the SDK has to communicate with.
 </div>
 
-1. Go to your [Developer Platform][4] within Datadog and click on **+ New App** in the upper right.
-
-<img style="max-width:80%" alt="New App" src="https://user-images.githubusercontent.com/228230/137548671-c0c64c2e-e3cd-494b-990c-8dc8a90d4800.png">
+### Add your app to the Developer Platform
 
-5. Enter a name for your application. Choose a distinct name so that you can differentiate it from any other apps you may be creating.
+1. Go to your [Developer Platform][4] within Datadog and click on **+ New App**.
```
Comment (alai97) [inline docs/en/getting-started.md]:
```suggestion
1. Navigate to [**Integrations** > **Developer Platform**][4] and click **+ New App**.
```

File: docs/en/getting-started.md [hunk 4]
```diff
@@ -19,97 +19,88 @@ Datadog Apps enable developers to extend the native functionality of Datadog thr
...
+   <img style="max-width:80%" alt="Add a new app to the developer platform" src="https://user-images.githubusercontent.com/228230/137548671-c0c64c2e-e3cd-494b-990c-8dc8a90d4800.png">
 
-6. You are then presented with the dashboard for your new application. You can further change your app name here, give your app a more detailed description, or change its icon.
+1. Enter a unique name for your application.
 
-<img style="max-width:80%" alt="App Edit Basic Information" src="https://user-images.githubusercontent.com/17037651/163401812-d21a9d3a-e73f-49b0-bda4-e7c447295784.png">
+1. (Optional) Once you're presented with the dashboard for your new application, you can change your app name, give the app a more detailed description, or change its icon.
```
Comment (alai97) [inline docs/en/getting-started.md]:
```suggestion
1. Optionally, once you're presented with the dashboard for your new application, you can change the app name, give the app a more detailed description, or change its icon.
```

File: docs/en/getting-started.md [hunk 5]
```diff
@@ -19,97 +19,88 @@ Datadog Apps enable developers to extend the native functionality of Datadog thr
...
+1. Once you're presented with more options for your app, change the **Root URL** and **Debug Mode Root URL** to match the `localhost` version of the widget you're running. The main controller path is `/widget`. These URL values will change as you build your application and begin to host it on your own infrastructure.
 
-3. Turn the toggle
-to ‘Dashboard Custom Widget’ on (you may need to scroll the center panel down a bit to see it). This generates JSON on the right hand side.
+1. Turn the toggle to **Dashboard Custom Widget** on. This generates JSON for the app.
+   
+   <img style="max-width:80%" alt="App Edit UI Extensions" src="https://user-images.githubusercontent.com/17037651/163402086-a3afbecd-c9c0-4608-bb91-6cb5391fec93.png">
 
-<img style="max-width:80%" alt="App Edit UI Extensions" src="https://user-images.githubusercontent.com/17037651/163402086-a3afbecd-c9c0-4608-bb91-6cb5391fec93.png">
+   Note that this JSON contains a value called `Your first widget`. This is the name of your widget as it appears in the menu to add to your Dashboards.
```
Comment (alai97) [inline docs/en/getting-started.md]:
```suggestion
   In this example, the JSON output contains a value called `Your first widget`. This is the name of your widget as it appears in the menu to add to your Dashboards.
```

File: docs/en/getting-started.md [hunk 6]
```diff
@@ -19,97 +19,88 @@ Datadog Apps enable developers to extend the native functionality of Datadog thr
...
 
-Note that this JSON contains a value called `Your first widget`. This is the name of your widget as it appears in the menu to add to your Dashboards.
 
-4. Navigate to your dashboard and add a widget.
+1. The **Custom Widgets** section is at the bottom of the sidebar. Find your widget in the list and add it to your dashboard.
 
-<img style="max-width:80%" alt="Dashboard add widget" src="https://user-images.githubusercontent.com/228230/137550297-3f98c5e0-0826-4109-b6e4-bf6dd1209aa2.png">
+   <img style="max-width:80%" alt="Add your widget from the custom widgets section" src="https://user-images.githubusercontent.com/228230/137550380-7b9b222d-c848-4d17-9060-cd0345780a11.png">
 
+1. A preview of your new widget appears, along with some options. Scroll down and click *Done* to add it to your dashboard.
```
Comment (alai97) [inline docs/en/getting-started.md]:
```suggestion
1. A preview of your new widget appears, along with some options. Scroll down and click **Done** to add it to your dashboard.
```

File: docs/en/getting-started.md [hunk 7]
```diff
@@ -19,97 +19,88 @@ Datadog Apps enable developers to extend the native functionality of Datadog thr
...
-6. A preview of your new widget appears, along with some options. Scroll down in the modal and click *Done* to add it to your dashboard.
-
-<img style="max-width:80%" alt="New Application" src="https://user-images.githubusercontent.com/228230/137550741-669f69c6-4a9b-4253-afc4-be3257a1084e.png">
-
-<img style="max-width:80%" alt="New Application 2" src="https://user-images.githubusercontent.com/228230/137550757-96bce01d-2ec4-4c0f-b045-e18b756e52df.png">
-
-
-
-To build your application, run `yarn build` in your terminal. Then move your static generated site to the hosting of your choice and update the URLs in the app settings.
+1. To build your application, run `yarn build` in your terminal. Then move your static generated site to the hosting of your choice and update the URLs in the app settings.
```
Comment (alai97) [inline docs/en/getting-started.md]:
```suggestion
1. To build your application, run `yarn build` in your terminal. Then, move your static generated site to the hosting platform of your choice and update the URLs in the app settings.
```

**Review by alai97 — APPROVED** [review]
Looks good, some copy suggestions!

**Review by enbashi — APPROVED** [review]
